src/main/webapp/resources/js/components/OutputFiles/JsonObjectOutputWrapper.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Component to render a div for JSON object output
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { SPACE_XS } from "../../styles/spacing";
+import { grey4 } from "../../styles/colors";
+import styled from "styled-components";
+
+/**
+ * Stateless UI component for displaying a div for json object output
+ *
+ * @param {number} height - Height of div. Default is 300px
+ * @param {string} description - Optional description
+ * @param {object} children - content to display in div
+ * @param {object} props - any other attributes to add to JsonOutputWrapper
+ *
+ * @returns {Element} - Returns a pre component
+ */
+
+export function JsonObjectOutputWrapper({ height = 300, children, ...props }) {
+  const Wrapper = styled.pre`
+    height: ${height}px;
+    margin-bottom: ${SPACE_XS};
+    border: solid 1px ${grey4};
+    background-color: #ffffff;
```

Comment:
Use the variable `grey1` instead of the hex code

Reply:
You also use `border: solid 1px ${grey4};` maybe we should make another style javascript file for borders so we can have standard borders.

Reply:
Updated in 5af0b378509161b98e4718f05132265df8b225c3